src/Actions/TimeoutTransitionAction.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Symbiote\AdvancedWorkflow\Actions;
+
+use Symbiote\AdvancedWorkflow\DataObjects\WorkflowAction;
+use Symbiote\AdvancedWorkflow\DataObjects\WorkflowInstance;
+use SilverStripe\Forms\DropdownField;
+use SilverStripe\Forms\FieldGroup;
+use SilverStripe\Forms\LiteralField;
+use SilverStripe\Forms\NumericField;
+use SilverStripe\ORM\FieldType\DBDatetime;
+
+class TimeoutTransitionAction extends WorkflowAction
+{
+    private static $db = [
+        'TimeoutCount' => 'Int',
+        'TimeoutIncrement' => 'Enum("Hours,Days,Weeks,Months,Years")',
+    ];
+
+    private static $has_one = [
+        'TimeoutTransition' => WorkflowTransition::class . '.Action'
+    ];
+
+    private static $icon = 'symbiote/silverstripe-advancedworkflow:images/transition.png';
+    private static $table_name = 'TimeoutTransitionAction';
+    private static $instance_class = TimeoutTransitionInstance::class;
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+
+        // increments
+        $inc = [];
+        preg_match('/Enum\(\"(.*)\"\)/', static::$db['TimeoutIncrement'], $inc);
```

Comment:
For future ref, you can access these values as

$increments = $this->dbObject('TimeoutIncrement')->enumValues()

Reply:
Fixed.